contracts/tgrade-validator-voting/src/msg.rs

```diff
@@ -101,6 +100,16 @@ pub enum ValidatorProposal {
         /// The contract address to be cleared
         contract: String,
     },
+    ClientUpdate {
```

Comment:
Here too, a proper `Ibc` prefix would be nice.

Reply:
It is very important to not have this mixed with a chain upgrade.
```suggestion
    IBCClientUpdate {
```
IBC is an acronym and uppercase

Reply:
@alpe in CamelCase acronyms are done with capitalizing only first letter. I already had issues with tools using it this common way. Same thing as we tend to name types as `TcpClient`, `HttpRequest` and so. So if it is to be prefixed, it would be actually `IbcClientUpdate`.